src/lib/Libpython/pbs_python_svr_size_type.c

```diff
@@ -587,20 +587,13 @@ pps_size_number_methods_subtract(PyObject *left, PyObject *right)
 
 		if (rc != 0)
 			goto QUIT;
-#ifdef NAS /* localmod 005 */
```

Comment:
why did you remove this NAS mod?

Reply:
NAS mod 5 has to do with removing warnings from the Intel compiler.  I didn't remove their code, I removed ours.  Since the intel compiler is a bit more pedantic than gcc, following what it wants should be fine for gcc.  I removed the mod because it cleaned up the code a bit.